test/jsonld/test_nested_arrays.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: UTF-8 -*-
+
+from rdflib import Graph, Literal, URIRef
+from rdflib.collection import Collection
+from rdflib.plugin import Parser, register
+
+register("json-ld", Parser, "rdflib.plugins.parsers.jsonld", "JsonLDParser")
+register("application/ld+json", Parser, "rdflib.plugins.parsers.jsonld", "JsonLDParser")
```

Comment:
I don't think this is needed, I will check though:

```suggestion
```

Reply:
You may be right (I tried to copy the structure of the other tests)

Reply:
This is fixed now.